docs/spark-connector/spark-catalog-paimon.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Spark connector Paimon catalog"
+slug: /spark-connector/spark-catalog-paimon
+keyword: spark connector paimon catalog
+license: "This software is licensed under the Apache License version 2."
+---
+
+The Apache Gravitino Spark connector offers the capability to read and write Paimon tables, with the metadata managed by the Gravitino server. To enable the use of the Paimon catalog within the Spark connector now, you must set download [Paimon Spark runtime jar](https://paimon.apache.org/docs/0.8/spark/quick-start/#preparation) to Spark classpath.
+
+## Capabilities
```

Comment:
could you keep the style as `spark-catalog-hive` or `spark-catalog-iceberg`?

Reply:
ok

Reply:
@FANNG1 Do you mean titles?

Reply:
yes

Reply:
updated cc @FANNG1